packages/components/table/src/TableBody/TableBody.tsx

```diff
@@ -7,7 +7,7 @@ import {
 } from '@contentful/f36-core';
 
 export type TableBodyInternalProps = CommonProps & {
-  children: ReactElement | ReactElement[];
+  children?: ReactNode;
```

Comment:
should it be nullable tho?

Reply:
for conditional rendering might be useful?

Reply:
like this
![Uploading CleanShot 2023-05-31 at 14.19.09@2x.png…]()

Reply:
Do you mean adding a condition as the children?? because even like that if you had `{someCondition && <div />}` it would still be considered as having a children, and it is a ReactNode 🤔
Or is there a case where we would not pass any children to the Table.Body??

Reply:
I guess there is no such case, that's why I rolled back to required children, thanks for raising it 👍